content/en/docs/concepts/extend-kubernetes/compute-storage-net/_index.md

```diff
@@ -39,6 +39,6 @@ fabric that links Pods together.
   Your Kubernetes cluster needs a _network plugin_ in order to have a working Pod network
   and to support other aspects of the Kubernetes network model.
 
-  Kubernetes {{< skew currentVersion >}} is compatible with {{< glossary_tooltip text="CNI" term_id="cni" >}}
+  Kubernetes {{< skew currentPatchVersion >}} is compatible with {{< glossary_tooltip text="CNI" term_id="cni" >}}
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.